cmake/config/config_armhf-linux.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015-2016 Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(CMakeForceCompiler)
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR armhf)
+
+SET(CMAKE_C_COMPILER   gcc)
```

Comment:
We cannot build armhf-linux on x86 linux host using above added config. It could make developers confused since most configs are for x86 linux host. What about setting `CMAKE_C_COMPILER` based on identified host name like

    if (${ARCHITECTURE} STREQUAL "x86_64" or ${ARCHITECTURE} STREQUAL "i686")
        set(CMAKE_C_COMPILER arm-linux-gnueabihf-gcc)
    endif()
    if (${ARCHITECTURE} STREQUAL "arm???")
        set(CMAKE_C_COMPILER gcc)
    endif()